lib/dradis/plugins/configurable.rb

```diff
@@ -3,12 +3,17 @@ module Configurable
     extend ActiveSupport::Concern
 
     module ClassMethods
-      delegate :settings, to: :instance
+      delegate :encrypted_settings, :settings, to: :instance
 
       def settings_namespace
         @settings_namespace || plugin_name
       end
 
+      def addon_encrypted_settings(namespace = nil, &block)
+        @settings_namespace = namespace if namespace
```

Comment:
```suggestion
        @settings_namespace = namespace.presence
```

Reply:
I'm not sure we need this `@settings_namespace = namespace` works just as good actually.

See https://github.com/dradis/dradis-plugins/pull/71/commits/1b77c9cdce7ff6f6de6b16ad35ede8dbb9852008